content/v2.0/reference/api/client-libraries/go.md

```diff
@@ -1,12 +1,183 @@
 ---
-title: InfluxDB Go client library
+title: Go client library
 list_title: Go
-description: Use the Go client library to interact with InfluxDB.
-external_url: https://github.com/influxdata/influxdb-client-go
+description: >
+  Use the Go client library to interact with InfluxDB.
 menu:
   v2_0_ref:
     name: Go
     parent: Client libraries
-    url: https://github.com/influxdata/influxdb-client-go
+v2.0/tags: [client libraries, Go]
+aliases:
+  - /v2.0/reference/api/client-libraries/go-cl-guide/
 weight: 201
 ---
+
+Use the [InfluxDB Go client library](https://github.com/influxdata/influxdb-client-go) to integrate InfluxDB into Go scripts and applications.
+
+This guide presumes some familiarity with Go and InfluxDB.
+If just getting started, see [Getting started with InfluxDB](/v2.0/get-started/).
+
+## Before you begin
+
+1. Go 1.3 or later is required. 
```

Comment:
1. [Install Go 1.3 or later](https://golang.org/doc/install)

_Raw markdown for easy copy and paste:_
```
1. [Install Go 1.3 or later](https://golang.org/doc/install)
```